pages/index.js

```diff
@@ -12,7 +12,7 @@ export default function Home() {
       </Head>
 
       <main className={styles.main}>
-        <h1 className={styles.title}>
+        <h1 className={`${styles.title} ${styles.nadpis}`}>
```

Comment:
musime ty classy psat takhle a nemuzem davat className="nadpis". to je vec toho sass package.

taky si vsimni, že musime psat className misto class, ktery se pise v html. To proto, ze ikdyz to tak vypada, tak toto neni html, ale jsx a nakonec se to prekompiluje do javascriptu.

Reply:
Ok a jsx už je jakoby z Reactu?

Reply:
jo je to react vec